build/linux/installer/conf/fluent-bit-common.conf

```diff
@@ -84,11 +84,3 @@
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
-
-[OUTPUT]
-    Name                            oms
-    Alias                           oms_input_plugin_output
-    EnableTelemetry                 true
```

Comment:
what's the reason for removing this?

Reply:
everything goes in one single output plugin and we route to appropriate endpoints internally. I found that this was more performant as compared to running two output oms plugins